translations/compfest-11-penyisihan/compfest-11-penyisihan-menggabungkan-monster.html

```diff
@@ -0,0 +1,96 @@
+<!-- Title : Merging Monster -->
+
+<h3> Description </h3>
+
+<p>
+	Mr. Chanek is a well known scientist in the Nation of Chanek. Mr. Chanek is researching about monsters which often attack the Nation of Chanek.
+</p>
+
+<p>
+	Mr. Chanek has $N$ experimental monsters lining from left to right and numbered from $1$ to $N$. The $i$-th monster has power $A_i$.
+
+	Mr. Chanek would like to do $M$ mergings. Mr. Chanek can choose a monster and merge it with the monster directly to the right of the former monster.
```

Comment:
Missing one sentence from the original statement

```suggestion
	Mr. Chanek would like to do an experiment by merging these monsters. Mr. Chanek would like to do $M$ mergings. Mr. Chanek can choose a monster and merge it with the monster directly to the right of the former monster.
```